workspace/codespaces/codespaces.py

```diff
@@ -0,0 +1,96 @@
+import collections
```

Comment:
It's not obvious from the file path `workspace/codespaces/codespaces.py` what the module's purpose is. Can there be a brief module docstring explaining that please?

Reply:
Alas, BennettBot stores this kind of information in `bennettbot/job_configs.py`. I will duplicate that information in a module docstring, with the caveat that the architecture could be improved.